<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportingTab</class>
 <widget class="QWidget" name="ReportingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Fixer</string>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="templateGroupBox">
     <property name="title">
      <string>Template Configuration</string>
     </property>
     <layout class="QHBoxLayout" name="templateLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="templateLabel">
        <property name="text">
         <string>Template File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="template_path_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select template Excel file...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browse_template_btn">
        <property name="text">
         <string>Browse...</string>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sourceGroupBox">
     <property name="title">
      <string>Source Report</string>
     </property>
     <layout class="QVBoxLayout" name="sourceLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="sourceFileLayout">
        <item>
         <widget class="QLabel" name="sourceLabel">
          <property name="text">
           <string>Source File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="source_path_edit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Drop Excel file here or click Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browse_source_btn">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="source_info_label">
        <property name="text">
         <string>No source file loaded</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #666; padding: 2px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputGroupBox">
     <property name="title">
      <string>Output Settings</string>
     </property>
     <layout class="QHBoxLayout" name="outputLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="customerLabel">
        <property name="text">
         <string>Customer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="customer_combo">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Select customer...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="outputSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewGroupBox">
     <property name="title">
      <string>Column Mapping Preview</string>
     </property>
     <layout class="QVBoxLayout" name="previewLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTableWidget" name="preview_table">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Column Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Action</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mapping_status_label">
        <property name="text">
         <string>Load template and source files to see column mapping</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #666;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="fix_export_btn">
       <property name="text">
        <string>Fix &amp;&amp; Export Report</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold; padding: 8px 16px;</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_output_btn">
       <property name="text">
        <string>Open Output Folder</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="actionSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="statusGroupBox">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="statusLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTextEdit" name="status_text">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f5f5f5; font-family: monospace;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
